### ISSUE TITLE
No sound from JamesDSP on Debian12

### ISSUE THREAD

**Issue by Homeswinghome** [Audio4Linux/JDSP4Linux#195]
Hello,
I just intalled JamesDSP on an old laptop running Debian12. Installation was first done with Nix package manager. As it didn't work, I remove it and installed JamesDSP via Flatpak. It worked fine on a first test with sound from Firefox/Youtube but stopped working when I closed Firefox to use an mp3 with Quolibet. Helvum shown the audio flux was correctly wired through JamesDSP. Not being an expert, it seems the flux shown by Helvum were in blue when it ran fine but didn't get the same color when it didn't work properly.
This problem seems already reported, maybe here in #164 but I haven't find (or understood) a fix.
Thank you for your help

**Comment by Homeswinghome** [comment]
Hello,
With my son's help, we came to the hypothesis it is a problem of compatibility between the Pipewire version expected by JamesDSP and the Pipewire version provided by Debian12.
JamesDSP is now working with a Pipewire 1.0.7 instead of the 0.3.65 (+ patch?)
We uninstalled the Debian package and replaced it by the one provided by the NIX package manager.
An other method would have been to open the system to Debian 12 backport which has a 1.0.7 Pipewire.
As Pipewire 1.0.x is not already available in the stable distributions like Debian or Ubuntu, some information about it might help... unless I didn't read carefully enough the instructions...
